Package.swift

```diff
@@ -7,14 +7,21 @@ let package = Package(
         .macOS(.v11)
     ],
     products: [
-        .library(name: "SignalRClient", targets: ["SignalRClient"]),
+        .library(name: "SignalRClient", targets: ["SignalRClient"])
     ],
     dependencies: [
+        .package(
+            url: "https://github.com/inaka/EventSource.git", .branch("master")
```

Comment:
Use a tag version instead of master to avoid any breaking?

Reply:
We can implement the `SSE` protocol by ourself later.